cranetoolbox/importTools/transform.py

```diff
@@ -84,11 +95,13 @@ def write_tweets_by_chunk(lines, csv_output_path: str, opts: TransformationOptio
             filtered_chunk, failure_count = filter_lighten_chunk(chunk, opts)
             parse_failure_count += failure_count
             line_count += len(filtered_chunk)
-            csv.writer(csv_file, quoting=csv.QUOTE_MINIMAL).writerows(filtered_chunk)
+            csv.writer(csv_file, quoting=csv.QUOTE_MINIMAL).writerows(
+                filtered_chunk)
     return line_count, parse_failure_count
 
 
-def filter_lighten_chunk(chunk, opts: TransformationOptions) -> (List[dict], int):
+def filter_lighten_chunk(chunk, opts: TransformationOptions) -> (
+List[dict], int):
```

Comment:
Isn't there some tab spaces missing here?

Reply:
Where? this line was formatted by pycharm plugin, but i can revert.

Reply:
I would have thought line 104 should be indented because that's the way other function defs are when they run over the character limit, but maybe I'm wrong

Reply:
Like this?
![image](https://user-images.githubusercontent.com/33736256/95134481-537a8200-0739-11eb-8042-28ad25731476.png)

Reply:
@LaChapeliere ?

Reply:
No (sorry, didn't see the notification), like this (I don't know how many tabs there should be):
`def filter_lighten_chunk(chunk, opts: TransformationOptions) -> (
                             List[dict], int):`

Reply:
But really, it doesn't matter that much, so let's not block the PR because of it. I'll trust your linter :)